lib/src/barback/asset_environment.dart

```diff
@@ -186,24 +193,6 @@ class AssetEnvironment {
 
     var isRootPackage = package.name == rootPackage.name;
     switch (compiler) {
-      case Compiler.dartDevc:
-        var firstPhase = <dynamic>[new ModuleConfigTransformer()];
-        var lastPhase = <dynamic>[
-          new DartDevcModuleTransformer(mode,
-              environmentConstants: environmentConstants)
-        ];
-        if (isRootPackage) {
-          firstPhase.add(new DartDevcResourceTransformer());
-          lastPhase.add(new DartDevcBootstrapTransformer(mode));
-        }
-
-        transformers.addAll([
-          firstPhase,
-          [new UnlinkedSummaryTransformer()],
-          [new LinkedSummaryTransformer()],
-          lastPhase,
-        ]);
-        break;
```

Comment:
Now that the AggregateTransformer phases are gone, are there any type annotations where you had to use dynamic to handle `Transformer | AggregateTransformer` that you can switch back to being precisely typed?